.github/workflows/publish_pytorch_dev_docker.yml

```diff
@@ -42,6 +44,10 @@ jobs:
           registry: ${{ env.REGISTRY }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set nightly version
+        id: nightly_version
+        run: echo "VERSION=${{ env.BASE_VERSION }}rc$(date +'%Y%m%d')" >> "$GITHUB_ENV"
```

Comment:
Where is this `env.BASE_VERSION` coming from? Should we load [`version.json`](https://github.com/ROCm/TheRock/blob/main/version.json) somehow? Maybe add a `compute_nightly_version.py` script and put the `rc` and `date` code in that script? Such a script could then set environment variables or outputs for this action to use.

Reply:
I am considering now the json file. `BASE_VERSION=$(jq -r '.["rocm-version"]' version.json)`

Reply:
Do you still considering a `compute_nightly_version.py` for that?